theories/dune

```diff
@@ -0,0 +1,38 @@
+(coq.theory
+ (name Coq)
+ (package coq)
+ (synopsis "Coq's Standard Library")
+ (flags -q)
+ ; (mode native)
+ (boot)
+ ; (per_file
+ ;  (Init/*.v -> -boot))
```

Comment:
Why keeping commented out lines?

Reply:
Mainly as a reminder to update this once dune has per_file flags, I'll add a comment.